recipes/dmri-amico/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dmri-amico" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "08fe9fe14554656fcfe9862dcdbb40f5cf9fc1cb9af5685435190110b83f1902" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - setup.py.patch
```

Comment:
Can you add a comment on why this patch is needed and/or if it was upstreamed?

Reply:
@ocefpaf Sure - there is an [open PR](https://github.com/daducci/AMICO/pull/179) upstream regarding this, and I've added a comment in `meta.yaml` explaining why it is needed for this version.